config/manager/manager.yaml

```diff
@@ -13,17 +13,20 @@ metadata:
   namespace: system
   labels:
     control-plane: controller-manager
+    operation: webhook
```

Comment:
should we namespace the labels (`gatekeeper.sh/operation`) to make sure we don't overlap with others' labels?